maintainers/maintainer-list.nix

```diff
@@ -960,11 +960,7 @@
     githubId = 16022854;
```

Comment:
Please revert all formatting changes in this file

Reply:
Thanks, missed that.